frame/sgx-hello-world/src/tests.rs

```diff
@@ -72,40 +81,110 @@ impl frame_system::Trait for Test {
 }
 
 type Extrinsic = TestXt<Call<Test>, ()>;
-type SubmitTransaction = frame_system::offchain::TransactionSubmitter<
-	crypto::Public,
-	Test,
-	Extrinsic
->;
-
-impl frame_system::offchain::CreateTransaction<Test, Extrinsic> for Test {
-	type Public = sp_core::sr25519::Public;
-	type Signature = sp_core::sr25519::Signature;
-
-	fn create_transaction<F: frame_system::offchain::Signer<Self::Public, Self::Signature>>(
-		call: <Extrinsic as ExtrinsicsT>::Call,
-		_public: Self::Public,
-		_account: <Test as frame_system::Trait>::AccountId,
-		nonce: <Test as frame_system::Trait>::Index,
-	) -> Option<(<Extrinsic as ExtrinsicsT>::Call, <Extrinsic as ExtrinsicsT>::SignaturePayload)> {
-		Some((call, (nonce, ())))
-	}
+type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::AccountId;
+
+impl frame_system::offchain::SigningTypes for Test {
+	type Public = <Signature as Verify>::Signer;
+	type Signature = Signature;
 }
 
-parameter_types! {
-	pub const GracePeriod: u64 = 5;
-	pub const UnsignedInterval: u64 = 128;
-	pub const UnsignedPriority: u64 = 1 << 20;
+impl<LocalCall> frame_system::offchain::SendTransactionTypes<LocalCall> for Test where
+	Call<Test>: From<LocalCall>,
+{
+	type OverarchingCall = Call<Test>;
+	type Extrinsic = Extrinsic;
+}
+
+impl<LocalCall> frame_system::offchain::CreateSignedTransaction<LocalCall> for Test where
+	Call<Test>: From<LocalCall>,
+{
+	fn create_transaction<C: frame_system::offchain::AppCrypto<Self::Public, Self::Signature>>(
+		call: Call<Test>,
+		_public: <Signature as Verify>::Signer,
+		_account: AccountId,
+		nonce: u64,
+	) -> Option<(Call<Test>, <Extrinsic as ExtrinsicT>::SignaturePayload)> {
+		Some((call, (nonce, ())))
+	}
 }
 
 impl Trait for Test {
 	type Event = ();
+	type AuthorityId = crypto::TestAuthId;
 	type Call = Call<Test>;
-	type SubmitSignedTransaction = SubmitTransaction;
-	type SubmitUnsignedTransaction = SubmitTransaction;
-	type GracePeriod = GracePeriod;
-	type UnsignedInterval = UnsignedInterval;
-	type UnsignedPriority = UnsignedPriority;
 }
 
-type Example = Module<Test>;
+type SgxTest = Module<Test>;
+
+fn setup_enclave_and_ias_responses(state: &mut testing::OffchainState) {
+	let fetch_quote_from_enclave = testing::PendingRequest {
+		method: "POST".into(),
+		uri: "https://myenclave_runs_here.example.com/quoting_report".into(),
+		response: Some(br#"123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here 123 whatever goes here "#.to_vec()),
+		sent: true,
+		// TODO: figure out what goes here
```

Comment:
The body is unused, it could just be an empty `vec!`. 

Or is the TODO referring to something else?